dss/events/chunkedtask/base.py

```diff
@@ -0,0 +1,43 @@
+import typing
+
+TaskStateType = typing.TypeVar('TaskStateType')
+RuntimeStateType = typing.TypeVar('RuntimeStateType')
+
+
+class Task(typing.Generic[TaskStateType]):
+    def run_one_unit(self) -> bool:
```

Comment:
In #323, I allow the task to return a value, so return values actually become significant.